### PR TITLE
What's New in Engine 3.1

### PR DESCRIPTION
**Pull request by micheles**
See https://github.com/gem/oq-engine/blob/whatsnew/doc/whats-new.md.

### CODE REVIEW

File: doc/whats-new.md [hunk 1]
```diff
@@ -1,307 +1,285 @@
...
-two realizations, computing mean, quantile-0.15 and quantile-0.85, i.e. three
-statistics).
+We added a warning in the case of a logic tree with more than 10,000
+realizations, suggesting the user to sample it because otherwise the
+calculation will likely run out of memory.
 
-We fixed the strange issue of very small negative numbers appearing in
-scenario_damage outputs: this happened due to rounding errors. Now
-the correct result (i.e. zeros) is stored.
+We earn the user when the source model contains duplicated source IDs,
```
Comment (mmpagani) [inline doc/whats-new.md]: we warn

File: doc/whats-new.md [hunk 2]
```diff
@@ -1,307 +1,285 @@
...
+suggesting to set the `optimize_same_id_sources` flag if the sources
+are really duplicated (the ID could be duplicated without the sources being so).
 
-We added a check in calculations reading the GMFs in CSV format: now
-the must be a single one realization in the input file.
+For scenario calculations, we added a che that the number of realizations
```
Comment (mmpagani) [inline doc/whats-new.md]: a check

File: doc/whats-new.md [hunk 3]
```diff
@@ -1,307 +1,285 @@
...
+We fixed the commands `oq to_shapefile` and `oq from_shapefile` by
+adding "YoungsCoppersmithMFD" and "arbitraryMFD" to the list of
+serializable magnitude-frequency distributions.
+
+Packaging
+----------
+
+For the first time we provide packages for Ubuntu 18.04 (Bionic).
+Old versions of Ubuntu (16.04 and 14.04) are still supported, and we
+keep providing packages for Red Hat distribution, as well as installers
```
Comment (daniviga) [inline doc/whats-new.md]: s/distribution/distributions/

**Comment by klunk386** [comment]
LGTM

**Pull request merged**